src/components/InventoryGroupDetail/InventoryGroupDetail.js

```diff
@@ -110,6 +108,14 @@ const InventoryGroupDetail = ({ groupId }) => {
       }
     }
   }, [data]);
+
+  if (
+    (fulfilled && data.total === 0) || // group does not exist
+    (rejected && error.status === 400) // group name not correct
```

Comment:
This is added as extra condition because HBI API responds with 400 if the group ID is not of the 4 UUID format.